### PR TITLE
refactor: Remove the redundant RelayerProcessor in monitor bot code

### PR DESCRIPTION
**Pull request by kevinuma**
Related PR: https://github.com/across-protocol/relayer-v2/pull/89

### CODE REVIEW

File: src/clients/MonitorClientHelper.ts [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-import { HubPoolClient } from "./HubPoolClient";
+import { getDeployedContract, getDeploymentBlockNumber, getSigner, winston } from "../utils";
+import { HubPoolClient, SpokePoolClient } from "../clients";
 import { constructClients, getSpokePoolSigners } from "../common";
+import { CHAIN_MAX_BLOCK_LOOKBACK } from "../common";
 
 export interface MonitorClients {
   hubPoolClient: HubPoolClient;
-  spokePools: { [chainId: number]: SpokePool };
+  spokePoolClients: { [chainId: number]: SpokePoolClient };
```
Comment (nicholaspai) [inline src/clients/MonitorClientHelper.ts:10]: Can import type `SpokePoolClientByChain` from "../interfaces" for this

Comment (kevinuma) [reply src/clients/MonitorClientHelper.ts]: Good suggestion thanks 👍 It was in RelayerConfigHelper so I moved it to interfaces.

File: src/relayer/RelayerClientHelper.ts [hunk 2]
```diff
@@ -3,16 +3,13 @@ import { Contract, getDeployedContract, getDeploymentBlockNumber, getSigner, Wal
...
 import { Clients, constructClients, updateClients, getSpokePoolSigners, updateSpokePoolClients } from "../common";
+import { SpokePoolClientsByChain } from "../interfaces";
 
 export interface RelayerClients extends Clients {
-  spokePoolClients: { [chainId: number]: SpokePoolClient };
+  spokePoolClients: SpokePoolClientsByChain;
   tokenClient: TokenClient;
 }
 
-export interface SpokePoolClientsByChain {
```
Comment (nicholaspai) [inline src/relayer/RelayerClientHelper.ts:12]: +1

File: src/clients/MonitorClientHelper.ts [hunk 3]
```diff
@@ -15,12 +17,27 @@ export async function constructMonitorClients(config: MonitorConfig, logger: win
...
+  const spokePoolClients = config.spokePoolChains.reduce((acc, chainId) => {
+    const spokePool = getDeployedContract("SpokePool", chainId, spokePoolSigners[chainId]) as SpokePool;
+    const startingBlock = Number(getDeploymentBlockNumber("SpokePool", chainId));
+    const spokePoolClient = new SpokePoolClient(
+      logger,
+      spokePool,
+      commonClients.configStoreClient,
+      Number(chainId),
+      {
+        fromBlock: startingBlock,
```
[the remark this answers is not in the document]
Comment (kevinuma) [reply src/clients/MonitorClientHelper.ts:29]: I was initially thinking about doing that but currently the monitor is not using any kind of lookback, so I just want to keep the status quo.

**Pull request closed without merging**